test/integration/cleanliness_test.rb

```diff
@@ -247,7 +247,7 @@ def assert_content(files)
         next if association.name == :audits # should never be destroyed
         next if association.options.key?(:through) # already cleaned up via through relation
         next if association.options.key?(:dependent) # already defined
-        next if association.is_a?(Doorkeeper::Application) && association.name == :authorized_tokens
+        next if association.name == :authorized_tokens # removed only when user is deleted; otherwise soft deleted
```

Comment:
seems a bit sus ... but if it works 🤷

Reply:
This was the exception we talked about for the Doorkeeper gem bump merged in yesterday. Not sure why the specs decided to grump about the syntax today, rather than yesterday when I was actually working on this 😅